src/blakesig.rs

```diff
@@ -1,3 +1,11 @@
+//! Hashing public keys using [BLAKE2b][]
+//!
+//! BLAKE2b is used to hash public keys to produce [Tezos] blockchain account identifiers.
+//!
+//! Functionality is also provided for serializing public keys as needed for Tezos addresses.
+//!
+//! [BLAKE2b]: https://blake2.net/
+//! [Tezos]: https://tezos.com/
```

Comment:
Maybe https://tezos.gitlab.io/ would be better, since it links directly to detailed dev docs on account types and blockchain specific topics?